src/motion/basic_target_follower_node.cpp

```diff
@@ -0,0 +1,18 @@
+#include "basic_target_follower.h"
+#include <ros/ros.h>
+#include <string>
+
+int main(int argc, char **argv) {
```

Comment:
Add some comments. What is this node doing? 
A line or 2 is good. Mention what are you getting from the subscriber and what will you be publishing.